aten/src/ATen/core/ivalue.cpp

```diff
@@ -64,6 +64,11 @@ bool operator==(const ivalue::Tuple& lhs, const ivalue::Tuple& rhs) {
              _fastEqualsForContainer);
 }
 
+std::ostream& operator<<(std::ostream& out, const ivalue::EnumHolder& v) {
+  out << v.qualifiedClassName() << "." << v.name();
+  return out;
+}
```

Comment:
Clang complained the declaration can't be found, it is because the definition is not in the proper namespace. This change moved the definition around.